src/motivating_scenario.md

```diff
@@ -84,14 +86,14 @@ enum Purpose:
     DUTY_OF_CARE
     STAFF_REPORT
 ```
-In general, we bundle all additional contextual information into the `Context` of the request.
+In general, we bundle all additional contextual information into the `Context` of the request.  
+These are the purposes we have defined as an example, but is not what might be included. 
 
 
 ### Policies
 
 With these, we hope to demonstrate Palisade is capable of applying complex record-level rules as part of the defined data-access policy, such as:
 * resource-level filtering - hide a resource `File /data/employee_file0.avro`
 * record-level filtering - hide the whole `Employee Alice` record
-* record-level masking - show the first half of `Alice`'s `Address`'s `PostCode` and hide the rest of the address
-* contextual rule application - show the full `BankDetails 1234-5678-90AB-CDEF` if the purpose of the request was `SALARY` and the user has completed the `PAYROLL_TRAINING_COURSE`, otherwise hide it
```

Comment:
Add this back in:
contextual rule application - show the full `BankDetails` if the purpose of the request was `SALARY` and the user has completed the `PAYROLL_TRAINING_COURSE`, otherwise hide it

Reply:
added